easybuild/tools/docs.py

```diff
@@ -707,6 +708,44 @@ def list_software_txt(software, detailed=False):
     return '\n'.join(lines)
 
 
+def gather_reverse_dependencies():
+    """
+    Gather reverse dependencies from installed modules.
+
+    :return: dictionary with reverse dependencies
+    """
+    reverse_dependencies = {}
+
+    module_avail_cmd = "module -t avail 2>&1 | grep -v ':$'"
+    module_purge_load_list_cmd = "module purge; module load %s; module -t list 2>&1 | grep -v %s"
+
+    available_modules, _ = run_cmd(module_avail_cmd)
+
+    for module in available_modules.splitlines():
+        dependencies, _ = run_cmd(module_purge_load_list_cmd % (module,module))
```

Comment:
@migueldiascosta probably a good idea to use `trace=False` here, otherwise you get very amusing output when using `eb --trace` (which is default for me)